src/data/metadata.csv

```diff
@@ -8243,14 +8243,54 @@ LT,LT,Lithuania,,,,,,,,-1
 LU,LU,Luxembourg,,,,,,,,-1
 LV,LV,Latvia,,,,,,,,-1
 LY,LY,Libya,,,,,,,,-1
-LY_BA,LY,Libya,BA,Benghazi,,,,,Benghazi,
-LY_JI,LY,Libya,JI,Jafara,,,,,Al-Jafara,
-LY_MS,LY,Libya,MS,Misrata District,,,,,Misrata,
-LY_SB,LY,Libya,SB,Sabha District,,,,,Sabha,
-LY_ST,LY,Libya,ST,Sabratha District,,,,,Sabratha,
-LY_TB,LY,Libya,TB,Tripoli District,,,,,Tripoli,
-LY_WS,LY,Libya,WS,Wadi al Shatii District,,,,,Wadi al Shati,
-LY_ZL,LY,Libya,ZL,Zliten District,,,,,Zliten,
+LY_BA,LY,Libya,,,BA,Benghazi,,,,
+LY_BU,LY,Libya,,,BU,Butnan,,,,
+LY_BW,LY,Libya,,,BW,Bani Walid,,,,
+LY_DR,LY,Libya,,,DR,Derna,,,,
+LY_GD,LY,Libya,,,GD,Ghadames,,,,
+LY_GH,LY,Libya,,,GH,Gharyan,,,,
+LY_GT,LY,Libya,,,GT,Ghat,,,,
+LY_JA,LY,Libya,,,JA,Jabal al Akhdar,,,,
```

Comment:
If these are all direct descendants of the country (they all appear to be districts of Libya) then they should be subregion1 and not subregion2.

Reply:
Done